src/util.rs

```diff
@@ -86,7 +86,7 @@ pub fn editor(original: String) -> Result<Value> {
     edit::edit_file(file.path())
         .map_err(|err| {
             log::debug!("{}", err);
-            log::error!("Error opening a text editor, pelase try --file");
+            log::error!("Error opening a text editor, please try using --filename");
```

Comment:
I would also print the file content to the console so the user can copy and paste it into the file

Reply:
Good Idea